atlp-rwanda/cabal-bn-fe
# #181339500-notification-panel

### What does this PR do?
 This PR implements notifications so that as a requester, manager, and travel admin on his/her notification panel
 
 ### description of the task
 - implements read notifications
 - increment notifications number icon
 - toast whenever there is a new notification
 ### How should this be tested manually?
- `git checkout feature/#181339500-notifications-panel` to go to this branch
- `yarn dev` to run the project
- log in as manager 
- on the other window log in as a requester: 
- click on notification icon to be able to see your notifications

### pivotal tracker task link
 [mark one/all notifications as read](https://www.pivotaltracker.com/story/show/181339512)
[Notification for requests update](https://www.pivotaltracker.com/story/show/181339500)

### Production link
[https://deploy-preview-21--barefoot-nomad-fe.netlify.app/](https://deploy-preview-21--barefoot-nomad-fe.netlify.app/)

### screenshot
<img width="1440" alt="Screen Shot 2022-06-03 at 12 22 43" src="https://user-images.githubusercontent.com/47631065/171836743-595b760b-07e4-4df8-9f7f-ccc4e682a761.png">
- mobile view
<img width="340" alt="Screen Shot 2022-06-06 at 10 27 13" src="https://user-images.githubusercontent.com/47631065/172125134-360f0f16-7988-442c-8de4-5b99b15d7f63.png">
